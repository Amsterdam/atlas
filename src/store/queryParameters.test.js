import queryParameters from './queryParameters';

// Todo: DP-6235
describe('QueryParameters', () => {
  it('should match the snapshot', () => {
    expect(queryParameters.result).toMatchSnapshot();
  });
});
