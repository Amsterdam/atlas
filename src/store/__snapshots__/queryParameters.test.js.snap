// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QueryParameters should match the snapshot 1`] = `
Object {
  "achtergrond": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "topografie",
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "baseLayer",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "topografie",
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "baseLayer",
      },
      "atlasRouter/DATA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "topografie",
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "baseLayer",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "topografie",
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "baseLayer",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "topografie",
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "baseLayer",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "topografie",
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "baseLayer",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "topografie",
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "baseLayer",
      },
    },
  },
  "bestand": Object {
    "routes": Object {
      "atlasRouter/CONSTRUCTION_FILE": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "",
        "encode": [Function],
        "reducerKey": "files",
        "selector": [Function],
        "stateKey": "fileName",
      },
    },
  },
  "categorie": Object {
    "routes": Object {
      "atlasRouter/DATA_QUERY_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": undefined,
        "encode": [Function],
        "reducerKey": "dataSearch",
        "selector": [Function],
        "stateKey": "category",
      },
    },
  },
  "center": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [
          52.3731081,
          4.8932945,
        ],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "viewCenter",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [
          52.3731081,
          4.8932945,
        ],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "viewCenter",
      },
      "atlasRouter/DATA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [
          52.3731081,
          4.8932945,
        ],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "viewCenter",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [
          52.3731081,
          4.8932945,
        ],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "viewCenter",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [
          52.3731081,
          4.8932945,
        ],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "viewCenter",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [
          52.3731081,
          4.8932945,
        ],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "viewCenter",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [
          52.3731081,
          4.8932945,
        ],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "viewCenter",
      },
    },
  },
  "detail-ref": Object {
    "routes": Object {
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "panorama",
        "selector": [Function],
        "stateKey": "detailReference",
      },
    },
  },
  "embed": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbed",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbed",
      },
      "atlasRouter/DATA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbed",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbed",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbed",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbed",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbed",
      },
    },
  },
  "embed-preview": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbedPreview",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbedPreview",
      },
      "atlasRouter/DATA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbedPreview",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbedPreview",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbedPreview",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbedPreview",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isEmbedPreview",
      },
    },
  },
  "filters": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {},
        "encode": [Function],
        "reducerKey": "filter",
        "selector": [Function],
        "stateKey": "filters",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {},
        "encode": [Function],
        "reducerKey": "filter",
        "selector": [Function],
        "stateKey": "filters",
      },
      "atlasRouter/DATASETS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {},
        "encode": [Function],
        "reducerKey": "filter",
        "selector": [Function],
        "stateKey": "filters",
      },
      "atlasRouter/DATA_QUERY_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {},
        "encode": [Function],
        "reducerKey": "filter",
        "selector": [Function],
        "stateKey": "filters",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {},
        "encode": [Function],
        "reducerKey": "filter",
        "selector": [Function],
        "stateKey": "filters",
      },
      "atlasRouter/SEARCH_DATASETS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {},
        "encode": [Function],
        "reducerKey": "filter",
        "selector": [Function],
        "stateKey": "filters",
      },
      "atlasRouter/SEARCH_DATA_CATEGORY": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {},
        "encode": [Function],
        "reducerKey": "filter",
        "selector": [Function],
        "stateKey": "filters",
      },
    },
  },
  "geo": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {
          "markers": undefined,
        },
        "encode": [Function],
        "reducerKey": "dataSelection",
        "selector": [Function],
        "stateKey": "geometryFilter",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {
          "markers": undefined,
        },
        "encode": [Function],
        "reducerKey": "dataSelection",
        "selector": [Function],
        "stateKey": "geometryFilter",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Object {
          "markers": undefined,
        },
        "encode": [Function],
        "reducerKey": "dataSelection",
        "selector": [Function],
        "stateKey": "geometryFilter",
      },
    },
  },
  "heading": Object {
    "routes": Object {
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 0,
        "encode": [Function],
        "reducerKey": "panorama",
        "selector": [Function],
        "stateKey": "heading",
      },
    },
  },
  "lagen": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/DATA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/DATA_QUERY_SEARCH": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/SEARCH_DATASETS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
      "atlasRouter/SEARCH_DATA_CATEGORY": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": Array [],
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "overlays",
      },
    },
  },
  "legenda": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "mapPanelActive",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "mapPanelActive",
      },
      "atlasRouter/DATA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "mapPanelActive",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "mapPanelActive",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "mapPanelActive",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "mapPanelActive",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "mapPanelActive",
      },
    },
  },
  "locatie": Object {
    "routes": Object {
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "dataSearch",
        "selector": [Function],
        "stateKey": "geoSearch",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "panorama",
        "selector": [Function],
        "stateKey": "location",
      },
    },
  },
  "marker": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "marker",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "marker",
      },
      "atlasRouter/DATA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "marker",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "marker",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "marker",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "marker",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": null,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "marker",
      },
    },
  },
  "modus": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "gesplitst",
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "viewMode",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "gesplitst",
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "viewMode",
      },
      "atlasRouter/DATA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "gesplitst",
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "viewMode",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "gesplitst",
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "viewMode",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "gesplitst",
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "viewMode",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "gesplitst",
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "viewMode",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "gesplitst",
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "viewMode",
      },
    },
  },
  "page-ref": Object {
    "routes": Object {
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": "",
        "encode": [Function],
        "reducerKey": "panorama",
        "selector": [Function],
        "stateKey": "pageReference",
      },
    },
  },
  "pagina": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": 1,
        "encode": [Function],
        "reducerKey": "dataSelection",
        "selector": [Function],
        "stateKey": "page",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": 1,
        "encode": [Function],
        "reducerKey": "dataSelection",
        "selector": [Function],
        "stateKey": "page",
      },
      "atlasRouter/DATASETS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": 1,
        "encode": [Function],
        "reducerKey": "datasets.datasetData",
        "selector": [Function],
        "stateKey": "page",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": 1,
        "encode": [Function],
        "reducerKey": "dataSelection",
        "selector": [Function],
        "stateKey": "page",
      },
    },
  },
  "pitch": Object {
    "routes": Object {
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 0,
        "encode": [Function],
        "reducerKey": "panorama",
        "selector": [Function],
        "stateKey": "pitch",
      },
    },
  },
  "print": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isPrintMode",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isPrintMode",
      },
      "atlasRouter/DATA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isPrintMode",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isPrintMode",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isPrintMode",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isPrintMode",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": false,
        "encode": [Function],
        "reducerKey": "ui",
        "selector": [Function],
        "stateKey": "isPrintMode",
      },
    },
  },
  "tags": Object {
    "routes": Object {
      "atlasRouter/PANORAMA": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": Array [
          "mission-bi",
        ],
        "encode": [Function],
        "reducerKey": "panorama",
        "selector": [Function],
        "stateKey": "tags",
      },
    },
  },
  "term": Object {
    "routes": Object {
      "atlasRouter/DATA_QUERY_SEARCH": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "",
        "encode": [Function],
        "reducerKey": "dataSearch",
        "selector": [Function],
        "stateKey": "query",
      },
      "atlasRouter/SEARCH_DATASETS": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "",
        "encode": [Function],
        "reducerKey": "dataSearch",
        "selector": [Function],
        "stateKey": "query",
      },
      "atlasRouter/SEARCH_DATA_CATEGORY": Object {
        "addHistory": true,
        "decode": [Function],
        "defaultValue": "",
        "encode": [Function],
        "reducerKey": "dataSearch",
        "selector": [Function],
        "stateKey": "query",
      },
    },
  },
  "zoom": Object {
    "routes": Object {
      "atlasRouter/ADDRESSES": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 11,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "zoom",
      },
      "atlasRouter/CADASTRAL_OBJECTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 11,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "zoom",
      },
      "atlasRouter/DATA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 11,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "zoom",
      },
      "atlasRouter/DATA_DETAIL": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 11,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "zoom",
      },
      "atlasRouter/DATA_GEO_SEARCH": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 11,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "zoom",
      },
      "atlasRouter/ESTABLISHMENTS": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 11,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "zoom",
      },
      "atlasRouter/PANORAMA": Object {
        "addHistory": false,
        "decode": [Function],
        "defaultValue": 11,
        "encode": [Function],
        "reducerKey": "map",
        "selector": [Function],
        "stateKey": "zoom",
      },
    },
  },
}
`;
